vendor-extra/JatsContentBundle/src/Handler/JatsContentHandler.php

```diff
@@ -46,11 +47,21 @@ public function handle(Element $documentElement, array $context) : array
 
         $contentHeader = $this->converter->convert($front, '@LiberoPatterns/content-header.html.twig', $context);
 
+        $content = [$contentHeader];
+
+        $tags = $this->converter->convert($front, '@LiberoPatterns/item-tags.html.twig', $context);
+        if ($tags->hasArgument('groups')) {
```

Comment:
Awkward to add into `JatsContentHandlerTest`, as the dumping converted won't set the real arguments...

Reply:
I have wondered whether these handlers should be replaced by emitting an event, and listeners then populate bits of the page (works with the page grid having a set of 'slots' that they use /cc @davidcmoulton).

Reply:
Both are HMVC-like, but that would be more separated (the single handler here being replaced by a number of event listeners).